data-prepper-plugins/elasticsearch/security.md

```diff
@@ -0,0 +1,49 @@
+# Elasticsearch Sink Security
+
+This document provides more details about the security settings of the sink.
+
+## AWS Elasticsearch Service
+
+Elasticsearch sink is capable of sending data to Amazon Elasticsearch domain that uses Identity and Access Management. The plugin uses the default credential chain. Run `aws configure` using the AWS CLI to set your credentials. 
```

Comment:
domains which use*

Also users don't need `aws configure` if they're running on EC2 w/ a service role attached. Might be worth calling it out somewhere in this doc (I can add it as a follow up, need to add to the deployment readme anyway).